compliance/azure/azure_low_cpu_usage/Azure CPU Below Utilization.pt

```diff
@@ -0,0 +1,256 @@
+name "Azure instances with low cpu average"
```

Comment:
rename to "Azure Idle Compute Instances"

Reply:
Ill rename it Azure Idle Compute Instances lol